src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2020,6 +2020,13 @@
 
   <Node Name="BoundInterpolatedStringHandlerPlaceholder" Base="BoundValuePlaceholderBase"/>
 
+  <!-- Type expression placeholder for the arguments to the constructor call for an interpolated string handler
```

Comment:
>Type expression [](http://example.com/codeflow?start=7&length=15)

"Type expression" sounds a lot like ```BoundTypeExpression```, but I believe this node doesn't represent a type reference. Because of that the comment feels confusing to me. I suggest saying "An expression" or "A typed expression" instead.